docs/modules/plugins/pages/plugin-azure-storage-account.adoc

```diff
@@ -169,6 +169,58 @@ When I find all blobs with name which $comparisonRule `$blobNameToMatch` in cont
 When I find all blobs with name which contains `json` in container `vividus` of storage account `testaccount` and save result to scenario variable `blobsCollection`
 ----
 
+==== Find blobs by filter
+
+Find blobs with names filtered by the specified rules in the container.
+At least one rule is required.
```

Comment:
does it make sense to highlight it? https://docs.antora.org/antora/2.3/asciidoc/admonitions/